src/components/ViewAgents/PopularWizards.tsx

```diff
@@ -6,21 +6,25 @@ import { toast } from "react-toastify";
 import { useGetAllAnalytics } from "hooks/reactQuery/wizards/useAnalytics";
 
 import Card from "./Card";
-import { useFetchPopularWizards } from "hooks/reactQuery/wizards/usePopularWizards";
+import { useFetchPublicWizards } from "hooks/reactQuery/wizards/usePublicWizards";
 import { WizardDetailsBasicWithTimeStamp } from "declarations/wizard_details/wizard_details.did";
-import classNames from "classnames";
+import SearchBarWizards from "components/ViewAgents/SearchBarWizards";
```

Comment:
```suggestion
import SearchBarWizards from "./SearchBarWizards";
```

Reply:
This was cause of the auto import update, changing it

Reply:
https://github.com/elna-ai/ELNA-DApp/pull/204/commits/7888d39dd7acb2b4b2f4d08148e3161aa6150055